### DISCUSSION TITLE
Remove Transparent Pixels from the Icons

### DISCUSSION THREAD

**Discussion by VoiditeDev** [ExperiencersInternational/auroraSuite#8] [Feature Requests]
The icons have Transparent Pixels. I agree it’s tricky to remove them all manually but a way to do this is by using [Pixelfix](https://github.com/Corecii/Transparent-Pixel-Fix)

![image](https://github.com/ExperiencersInternational/auroraSuite/assets/142426954/26903af8-aa08-4e4b-bd75-6028fc155372)
![image](https://github.com/ExperiencersInternational/auroraSuite/assets/142426954/62ec2e78-f2fd-4653-aa77-dae2ed8c889d)
![image](https://github.com/ExperiencersInternational/auroraSuite/assets/142426954/16f67bc7-1b7b-4ae5-b5de-5d57d7bdfd25)

> **NOTE**: THIS IS IMPORTED FROM THE DEVELOPER FORUMS

**Comment by ExperiencersInternational** [comment]
Working on this now, sorry for delay
